docs/rpc_protocol.md

```
@@ -0,0 +1,275 @@
+```json
+{
+  "kind": "stop",
+  "reason": "shutdown"
+}
+```
+
+## Request/Response Lifecycle
+
+### Simple Call Flow
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)